cutplace/interface.py

```diff
@@ -26,6 +26,7 @@
 import io
 import logging
 import os.path
+import sqlite3 as lite
```

Comment:
Just use `sqlite3`, you don't gain much from shortening it to `lite` but confuse others a lot.